.github/workflows/CI-docs.yml

```diff
@@ -30,15 +30,19 @@ jobs:
         shell: bash -l {0}
         run: |
           make -C docs/ html
+          exit_code=$?
+          if [ $exit_code -ne 0 ]; then
+            echo "Make command failed with exit code $exit_code"
```

Comment:
I agree it is good to include this 'if statement' here. Is it also possible to make it a github test that would fail in case we get an error in the make command?

Reply:
This is exactly what this if statement is doing. It fails if make exits with an error code. 

Have a look at https://github.com/gammasim/simtools/actions/runs/6823947507/job/18558879126 
I have put some non-ascii for this test into the code and the github action reported it.